packages/plugin-default-event-tracking-advanced-browser/src/default-event-tracking-advanced-plugin.ts

```diff
@@ -200,13 +200,23 @@ export const defaultEventTrackingAdvancedPlugin = (options: Options = {}): Brows
     }
     const addListener = (el: Element) => {
       if (shouldTrackEvent('click', el)) {
-        addEventListener(el, 'click', () => {
+        addEventListener(el, 'click', (event: Event) => {
+          // Limit to only the element that was clicked, not the propagated event.
+          /* istanbul ignore next */
+          if (event?.target != event?.currentTarget) {
```

Comment:
What about checking that the child nodes don't contain a match the CSS selector list?

```
 if (event?.currentTarget?.querySelector(cssSelectorsList)) {
```